neptune_tensorflow_keras/impl/__init__.py

```diff
@@ -92,23 +95,19 @@ class NeptuneCallback(Callback):
         You need to have Keras or Tensorflow 2 installed on your computer to use this module.
     """
 
-    def __init__(self, run: Run, base_namespace: Optional[str] = None):
+    def __init__(self, run: Run, base_namespace: str = "training"):
```

Comment:
I think we should mention in the PR description that there are BC-Breaking changes.

Reply:
Why do we even change it?

Reply:
@aniezurawski that's what's in the spec.

Reply:
> I think we should mention in the PR description that there are BC-Breaking changes.

And add some info to CHANGELOG.

Reply:
Done.